### DISCUSSION TITLE
The logged-on user doesn't have admin rights

### DISCUSSION THREAD

**Discussion by mmol67** [farag2/Sophia-Script-for-Windows#388] [General]
Hello! I have this problem.
My log in user is NOT administrator, but I opened the PowerShell window with administrator rights. I have PowerShell 2.0 activated.

Is this normal? Do I have to execute logged with my admin user?

**Comment by farag2** [comment]
Hi. What's your WIndows build? Also please upload a screenshot. :)

> My log in user is NOT administrator,

You have to be an admin by default. This is not the script restrictions, but Windows ones. This is how it works. YOu cannot bypass it.

>I have PowerShell 2.0 activated.

Yes, Windows (even modern ones) has it, but there is no need use it. PS 5.1 or 7.2 is much more modern. Also I advice to disable it (via Windows UI or script). All trojans uses it to be compatible with Windows 7. Feel free to ask anything. :)

**Reply by mmol67** [reply]
21H1 here. Using admin user, and it is working now. Thank you!
